src/braket/aws/aws_session.py

```diff
@@ -60,8 +60,12 @@ def __init__(self, boto_session=None, braket_client=None, config=None, default_b
             )
             self.braket_client = braket_client
         else:
-            self.boto_session = boto_session or boto3.Session()
-            self.braket_client = self.boto_session.client("braket", config=self._config)
+            self.boto_session = boto_session or boto3.Session(
+                region_name=os.environ.get("BRAKET_REGION")
```

Comment:
Do we want to use [AWS_REGION](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-region) here instead? This may be better if a customer is leaning on this env variable for other resources.

Reply:
Great call out, yes I agree.

Reply:
So it doesn't seem like AWS_REGION is in the [list of global env variables for boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#using-environment-variables), but I still agree with changing the name for parity/reuse